Pathfinder/2E/action-icons/readme.md

```diff
@@ -12,6 +12,15 @@ Ability to enter the Pathfinder 2e Action Icons into your notes.
 *⭓{ .Pathfinder }* - Free Action
 *⬲{ .Pathfinder }* - Reaction
 
+**Examples**
+![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/6f4d5183-8592-4f5f-9372-2adc065657a0)
```

Comment:
Similarly here, this is viewed as One line. Headers might also be appropriate here. Whichever ends up looking better.


```
**Examples**

 ![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/6f4d5183-8592-4f5f-9372-2adc065657a0)
```